compatibility-kit/ruby/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.feature
```

Comment:
When we create a new language module I think we should use the `.template` (add an `.rsync` file). If you need more specific `.gitignore`, add them in sub folders.

Reply:
indeed. I kept the one when this was under `cucumber-ruby/cck`, I'll use the template one (same for default.mk)